independent-publisher-connectors/TSheets Quickbooks Time/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "9da109344cdac6a322fcb020a4bb6536",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://rest.tsheets.com/api/v1/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://rest.tsheets.com/api/v1/grant"
+            },
+            "refreshUrl": {
+              "value": "https://rest.tsheets.com/api/v1/grant"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Artesian Software Technologies LLP",
```

Comment:
@artesianst can you please confirm if you own this API or not? if not please change the stachOwner property to the name of the service owner.